Volha_Ivanova/Module_1_Java/Java_Fundamentals/HomeWork1/src/com/epam/cdp/volha_ivanova/task3/Runner.java

```diff
@@ -0,0 +1,68 @@
+package com.epam.cdp.volha_ivanova.task3;
+
+import java.util.Scanner;
+
+public class Runner {                                       //Volha Ivanova
+	
+  public static void main(String[] args) {
+    boolean isRepeat = true;
+
+    while (isRepeat) {                                      //Printing user menu
+      System.out.println("Please, enter number of menu:");
+      System.out.println("1 - Summation");
+      System.out.println("2 - Subtraction");
+      System.out.println("3 - Multiplication");
+      System.out.println("4 - Division");
+      System.out.println("0 - Exit");
+      Scanner scanner = new Scanner(System.in);
+      
+      int action = scanner.nextInt();
+      int enteredValue1;
+      int enteredValue2;
+      
+          
+      switch (action) {                                     //Calling method according to user action
```

Comment:
I think you should refactor your switch-case block to comply DRY principle - do not repeat yourself (line 31-33)
1. check if action is 0
2. else read values and perform switch case